### DISCUSSION TITLE
In IGazeControl, how to block eyeball (convergence, tilt), only control neck?

### DISCUSSION THREAD

**Discussion by YANGGANG1971** [robotology/community#400] [Q&A] [answered]
http://www.yarp.it/classyarp_1_1dev_1_1IGazeControl.html#ab5e2d8e02496f95be7a01c932f18075b
I want to let the robot gaze at an object using the function below. By default, does this function control neck not control convergence, tilt?

> igaze->lookAtFixationPoint();

 If not, can yarp::dev::ICartesianControl::setLimits() set the range of eyeball's movement? In this function, there is a parameter - axis joint index (regardless it is actuated or not). How do know the corresponding joint index (e.g. tilt)?

**Comment by pattacini** [comment, marked as answer]
Hi @YANGGANG1971 

You should use the service [`yarp:dev::IGazeControl::blockEyes()`][1].
However, we can currently block only eyes vergence and pan, not the tilt.

[1]: http://www.yarp.it/classyarp_1_1dev_1_1IGazeControl.html#aa939635cf90acbb5e0f49bb114f86bc3
